CHANGELOG.md

```diff
@@ -1,10 +1,9 @@
 # Snyk Security Changelog
 
-## [2.12.2]
```

Comment:
We don't need a new version for each PR, just need to add changelog messages to the next version after the latest release

Reply:
Yeah, my fault...